### PR TITLE
Feature header

### PR DESCRIPTION
**Pull request by AaqibMushtaq**
Screenshot 2023-11-21 114712

### CODE REVIEW

File: images/home1-img 1.svg
Comment (UmmarHamid) [inline images/home1-img 1.svg:1]: file names should not include spaces.
Also try to give proper names to files like for this one you can use food-platter

Please do this for all images

File: index.html [hunk 1]
```diff
@@ -4,6 +4,51 @@
...
+            <p>
+              Fish Salad <br />
+              Fish Salad with Sweet <br />
+              Potatoes and Basil
+            </p>
+          </div>
+        </div>
+        <div class="right">
+          <div class="heading1"><p>#1 OF THIS WEEK</p></div>
+          <div class="heading2">
```
Comment (UmmarHamid) [inline index.html]: heading1, heading2 are not good examples of classnames.
Can you make them more sensible??

File: styles.css [hunk 2]
```diff
@@ -1 +1,77 @@
...
+.logo {
+  display: flex;
+  color: #fff;
+  justify-self: start;
+  align-items: center;
+}
+nav {
+  color: #fff;
+  display: flex;
+  text-align: center;
```
Comment (UmmarHamid) [inline styles.css:22]: Is this required?
can we use justify instead?

File: styles.css [hunk 3]
```diff
@@ -1 +1,77 @@
...
+  align-items: center;
+  gap: 48px;
+}
+header {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 26px;
```
Comment (UmmarHamid) [inline styles.css]: do we need a bottom padding as well??

File: styles.css [hunk 4]
```diff
@@ -1 +1,77 @@
...
+  font-size: 16px;
+  font-weight: 400;
+  width: 112px;
+  height: 41px;
+}
+nav {
+  display: flex;
+  display: inline-flex;
+  justify-content: space-evenly;
+}
```
Comment (UmmarHamid) [inline styles.css]: same selector used above, can we combine these?

File: styles.css [hunk 5]
```diff
@@ -1 +1,77 @@
...
+.right {
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+  margin-top: 32px;
+}
+
+.heading2 p {
+  /* display: flex; */
+  /* gap: 2px; */
```
Comment (UmmarHamid) [inline styles.css]: can we remove commented code?

**Review by UmmarHamid — CHANGES_REQUESTED** [review]

File: images/home1-img 1.svg
Comment (AaqibMushtaq) [reply images/home1-img 1.svg:1]: done

File: index.html (hunk 1 above)
Comment (AaqibMushtaq) [reply index.html]: done

File: styles.css (hunk 2 above)
Comment (AaqibMushtaq) [reply styles.css:22]: done

File: styles.css (hunk 3 above)
Comment (AaqibMushtaq) [reply styles.css]: yes sorry for that i just forgot  now its fixed

File: styles.css (hunk 4 above)
Comment (AaqibMushtaq) [reply styles.css]: done

File: styles.css (hunk 5 above)
Comment (AaqibMushtaq) [reply styles.css]: done

File: images/Food-Platter.svg
Comment (UmmarHamid) [inline images/Food-Platter.svg:1]: use smallcase for image names: food-platter.svg

File: index.html [hunk 6]
```diff
@@ -4,6 +4,51 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Healthy Resturant</title>
+    <link rel="stylesheet" href="styles.css" />
   </head>
-  <body></body>
+  <body>
+    <header>
+      <div class="logo">
+        <img src="/Images/hamburgerlogo.svg" alt="" />
```
Comment (UmmarHamid) [inline index.html]:
```suggestion
        <img src="/images/hamburgerlogo.svg" alt="Header Image" />
```

File: index.html [hunk 7]
```diff
@@ -4,6 +4,51 @@
...
+          <div>Menu</div>
+          <div>New</div>
+          <input class="support-button" type="button" value="Support" />
+        </nav>
+      </div>
+    </header>
+    <main>
+      <div class="hero-section">
+        <div class="left">
+          <img src="/Images/Food-Platter.svg" alt="" />
```
Comment (UmmarHamid) [inline index.html]: same as above

File: index.html [hunk 8]
```diff
@@ -4,6 +4,51 @@
...
+          </div>
+        </div>
+        <div class="right">
+          <div class="caption"><p>#1 OF THIS WEEK</p></div>
+          <div class="text">
+            <p>EAT</p>
+            <p>HEALTHY</p>
+            <p>BE HEALTHY</p>
+          </div>
+          <div class="discription">
```
Comment (UmmarHamid) [inline index.html]:
```suggestion
          <div class="description">
```

File: styles.css [hunk 9]
```diff
@@ -1 +1,78 @@
...
+  color: #fff;
+  display: flex;
+  text-align: center;
+  align-items: center;
+  gap: 48px;
+  display: flex;
+  display: inline-flex;
+  justify-content: space-evenly;
+  padding-left: 417px;
+}
```
Comment (UmmarHamid) [inline styles.css]:
```suggestion
nav {
  color: #fff;
  display: flex;
  align-items: center;
  gap: 48px;
  justify-content: space-evenly;
}
```

File: styles.css [hunk 10]
```diff
@@ -1 +1,78 @@
...
+nav {
+  color: #fff;
+  display: flex;
+  text-align: center;
+  align-items: center;
+  gap: 48px;
+  display: flex;
+  display: inline-flex;
+  justify-content: space-evenly;
+  padding-left: 417px;
```
Comment (UmmarHamid) [inline styles.css:27]: This should not be here!

File: styles.css [hunk 11]
```diff
@@ -1 +1,78 @@
...
+}
+.support-button {
+  background-color: #f4511f;
+  color: #fff;
+  border-radius: 24px;
+  padding: 9px 24px;
+  font-size: 16px;
+  font-weight: 400;
+  width: 112px;
+  height: 41px;
```
Comment (UmmarHamid) [inline styles.css]: can we avoid width/height and acheive size using padding and font size only?

**Review by UmmarHamid — CHANGES_REQUESTED** [review]

**Comment by UmmarHamid** [comment]
@AaqibMushtaq Can you please add screenshot of your latest work along with the PR in the description?

File: styles.css (hunk 11 above)
Comment (AaqibMushtaq) [reply styles.css]: done

File: styles.css (hunk 10 above)
Comment (AaqibMushtaq) [reply styles.css:27]: done

File: images/Food-Platter.svg
Comment (AaqibMushtaq) [reply images/Food-Platter.svg:1]: done

**Review by UmmarHamid — APPROVED** [review]
Looks good for now.
Great work and effort @AaqibMushtaq

**Pull request merged**